portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+$(document).ready(function() {
+    $("#bodynav li").click(function() {
+        $(this).addClass("active");
+        $(this).siblings().removeClass("active");
+        $("#headnav li").removeClass("active");
+    })
+    $("#headnav li").click(function() {
```

Comment:
why is behaviour different for this and above. Can this be combined by using some class?

Reply:
the list elements of bodynav class are supposed to be collapsed, while About me should remain there, that's why they are in different div